<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageLoadoutPaletteEditor</class>
 <widget class="QWidget" name="PageLoadoutPaletteEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>262</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QToolBox" name="toolBox">
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="loadout1">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>400</width>
        <height>69</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Loadout 1</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_16">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="LoadoutForm" name="loadout1Content" native="true">
         <property name="loadoutIndex" stdset="0">
          <UInt>0</UInt>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="loadout2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>400</width>
        <height>69</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Loadout 2</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_17">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="LoadoutForm" name="loadout2Content" native="true">
         <property name="loadoutIndex" stdset="0">
          <UInt>1</UInt>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="loadout3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>400</width>
        <height>69</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Loadout 3</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_18">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="LoadoutForm" name="loadout3Content" native="true">
         <property name="loadoutIndex" stdset="0">
          <UInt>2</UInt>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="loadout4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>400</width>
        <height>69</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Loadout 4</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_19">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="LoadoutForm" name="loadout4Content" native="true">
         <property name="loadoutIndex" stdset="0">
          <UInt>3</UInt>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="loadout5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>400</width>
        <height>69</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Loadout 5</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_20">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="LoadoutForm" name="loadout5Content" native="true">
         <property name="loadoutIndex" stdset="0">
          <UInt>4</UInt>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer_14">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>49</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LoadoutForm</class>
   <extends>QWidget</extends>
   <header>ui/LoadoutForm.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
